backend/src/server.ts

```diff
@@ -33,7 +33,7 @@ export const backend = onRequest({ region: "us-west1" }, app);
 
 export const beforecreated = beforeUserCreated((event) => {
   const user = event.data;
-  if (ENVIRONMENT === "production" && !user.email?.includes("unionstationhs.org")) {
+  if (ENVIRONMENT === "staging" && !user.email?.includes("@ucsd.edu")) {
```

Comment:
Can we revert this change?

```suggestion
  if (ENVIRONMENT === "production" && !user.email?.includes("unionstationhs.org")) {
```

Reply:
Reverted